core/tests/blocks.rs

```diff
@@ -0,0 +1,48 @@
+use unimarkup_commons::test_runner::{self, test_file};
+use unimarkup_core::Unimarkup;
+
+pub fn test_block_snapshots() -> Vec<libtest_mimic::Trial> {
```

Comment:
```suggestion
pub fn test_core_snapshots() -> Vec<libtest_mimic::Trial> {
```

See remark on the file.